examples/python-kernel-wrapper/README.md

```diff
@@ -0,0 +1,28 @@
+# Python Kernel Wrapper
+
+## Overview
+This examples aims to demonstrate how its possible to share cudf dataframes between Python and CUDA Kernels. This is useful if either an existing CUDA codebase is present to solve business logic or if teams would like the ability to handle certain tasks in Python and others in CUDA.
+
+Since cudf uses libcudf, dataframes that are created in Python are already present on the GPU memory and available to the C++ codebase. This makes it fairly straightforward to write a CUDA kernel for working with a dataframe column. In fact this is how libcudf processes dataframes in CUDA kernels within cudf itself, the only difference in this example is we are writing CUDA kernels that are not in the cudf codebase. The term User Defined Function (UDF) could be loosely used to describe what this example is demonstrating.
```

Comment:
```suggestion
Dataframes that are created in Python cuDF are already present in GPU memory and accessible to CUDA code. This makes it straightforward to write a CUDA kernel to work with a dataframe columns. In fact this is how libcudf processes dataframes in CUDA kernels; the only difference in this example is that we invoke CUDA kernels that exist outside the cuDF code base. The term User Defined Function (UDF) could be loosely used to describe what this example is demonstrating.
```
I would maybe drop the UDF sentence. UDFs tend to follow a well-defined interface and are defined outside cuDF but are applied to data frames or columns *by* cuDF. So this is loose enough to possibly cause confusion.